sass/includes/_card-grid.scss

```diff
@@ -2,7 +2,7 @@
     @extend .row;
     display: grid;
     
-    &--duo {
+    &__duo {
```

Comment:
Note: Refactoring to allow for BEM notation, I've added a modifier to one class so it makes sense to refactor the rest for consistency.